templates/applicantForm.phtml

```diff
@@ -149,9 +149,13 @@
                         <span class="input-group-addon">%</span>
                     </div>
 
-                    <label for="averageScore">Average of Aptitude/Attitude Score (%)</label>
+                    <label for="averageScore">Average of Aptitude/Attitude Score (%) <?php if ($applicant->getAverageScore()) {echo "[Edited by admin]";} ?> </label>
```

Comment:
Perhaps the brackets could be a different colour for greater readability by the user?